tools/Makefile.am

```diff
@@ -75,6 +75,7 @@ findif_SOURCES		= findif.c
 
 storage_mon_SOURCES	= storage_mon.c
 storage_mon_CFLAGS	= -D_GNU_SOURCE
+storage_mon_LDADD	= -lqb
```

Comment:
should go to the spec as well

Reply:
linking and cflags should be detected via PKG_CONFIG in configure.ac and propagated. No short cuts here.

Reply:
yep that too  - but what I had meant was BuildRequires: libqb-devel

Reply:
that was my point in the first place - sry for the confusion ;-)

Reply:
Regarding this point, what about the following changes?

```
diff --git a/configure.ac b/configure.ac
index 7b5faff5..9e0861b6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -621,6 +621,8 @@ PKG_CHECK_MODULES([GLIB], [$GPKGNAME])
 CPPFLAGS="$CPPFLAGS $GLIB_CFLAGS"
 LIBS="$LIBS $GLIB_LIBS"
 
+PKG_CHECK_MODULES([LIBQB], "libqb")
+
 dnl ========================================================================
 dnl Headers
 dnl ========================================================================
diff --git a/resource-agents.spec.in b/resource-agents.spec.in
index b974a693..829c17b3 100644
--- a/resource-agents.spec.in
+++ b/resource-agents.spec.in
@@ -55,7 +55,7 @@ Provides:     heartbeat-resources = %{version}
 BuildRequires: make
 BuildRequires: automake autoconf pkgconfig gcc
 BuildRequires: perl
-BuildRequires: libxslt glib2-devel
+BuildRequires: libxslt glib2-devel libqb-devel
 BuildRequires: systemd
 BuildRequires: which
 
diff --git a/tools/Makefile.am b/tools/Makefile.am
index 5c588796..8c96df98 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -75,7 +75,7 @@ findif_SOURCES                = findif.c
 
 storage_mon_SOURCES    = storage_mon.c
 storage_mon_CFLAGS     = -D_GNU_SOURCE
-storage_mon_LDADD      = -lqb
+storage_mon_LDADD      = ${LIBQB_LIBS}
 
 if BUILD_TICKLE
 halib_PROGRAMS         += tickle_tcp
```

Reply:
something like that, but you need to use LIBQB_CFLAGS as well.

Reply:
> something like that, but you need to use LIBQB_CFLAGS as well.
Okay!
thanks!